searxng/searxng
# Oscar theme: fix search input field on Safari

## What does this PR do?

Oscar theme: make sure search input field z-index is always 2 even when focused.

Code from https://github.com/tiekoetter/searxng/commit/c45611e5c6583ca01864c7c2ed8ae51bf7c3e6f3

## Why is this change important?

On Safari the input field disappears. 

## How to test this PR locally?

with Firefox, Chrome & Safari, check the search input field is displayed:
* when focused
* when the autocompletation are displayed
* when not focused

## Author's checklist



## Related issues

Close #323
